metrics/timer.js

```diff
@@ -13,10 +13,27 @@ var Timer = module.exports = function Timer() {
 }
 
 Timer.prototype.update = function(duration) {
+  // Manually adds a duration in ms to the histogram.
   this.histogram.update(duration);
   this.meter.mark();
 }
 
+Timer.prototype.time = function() {
+  // Starts the timer
+  var d = new Date();
```

Comment:
This is a minor thing, but It would be good to use [`process.hrtime`](https://nodejs.org/api/process.html#process_process_hrtime_time) instead of `Date` if available since it uses a monotonic clock and therefore it isn't affected by things like clock drift or the time being changed by somebody on the os.